sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -135,32 +135,62 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
+   * @group agg_funcs
+   * @since 1.3.0
+   */
+  @deprecated("Use approx_count_distinct()", "2.1.0")
```

Comment:
This is super minor - but `approx_count_distinct takes` arguments and the deprecation notice could maybe be improved a little by showing that (or at least not showing `()` - although looking at the current deprecation warnings we do appear to use both styles so matter of personal preference.